Packs/Druva/Integrations/Druva/Druva.yml

```diff
@@ -15,10 +15,67 @@ configuration:
   name: clientId
   type: 4
   required: true
+  defaultvalue: K7QYkjixOag7f2P0J/t2sgXq9NMsbzj/
```

Comment:
Is this default value should work for some users? If not, lets remove it

Reply:
default value is removed